core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -186,6 +186,25 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
     }
   }
 
+  def getRuntimeOptions(googleProjectId: String, bucketName: String)(implicit executionContext: ExecutionContext): Future[Option[JsValue]] = {
+    for {
+      regionOption <- googleServicesDAO.getRegionForRegionalBucket(bucketName)
+      runtimeOptions <- {
+        regionOption match {
+          case Some(region) =>
+            for {
+              zones <- googleServicesDAO.getComputeZonesForRegion(GoogleProjectId(googleProjectId), region)
+            } yield {
+              Option(JsObject("zones" -> JsString(zones.mkString(" "))))
+            }
```

Comment:
This looks more like it could be a simple `map` rather than a one-line `for` comprehension?